### PR TITLE
Adding environment setup in the installation page

### PR DESCRIPTION
**Pull request by dcharrezt**
This PR addresses Issue #493

### CODE REVIEW

File: docs/_pages/installation.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ Supported release is ROS-Melodic. It can be easily installed from official Debia
     ```bash
     sudo apt install ros-melodic-desktop-full
     ```
+    
+4. Environment setup
+    ```bash
+    echo "source /opt/ros/melodic/setup.bash" >> ~/.bashrc
+    source ~/.bashrc
+    ```
```
Comment (ojitmehta123) [inline docs/_pages/installation.md]:
```suggestion
5. Dependencies for building packages
    ```bash  
   sudo apt install python-rosdep python-rosinstall python-rosinstall-generator python-wstool build-essential  
   sudo apt install python-rosdep  
   sudo rosdep init  
   rosdep update  
    ```
For more information, refer [ros wiki](http://wiki.ros.org/)
```

File: docs/_pages/installation.md [hunk 2]
```diff
@@ -53,7 +53,20 @@ Supported release is ROS-Melodic. It can be easily installed from official Debia
...
+    source ~/.bashrc
+    ```    
+5. Dependencies for building packages
+    ```bash  
+   sudo apt install python-rosdep python-rosinstall python-rosinstall-generator python-wstool build-essential  
+   sudo apt install python-rosdep  
+   sudo rosdep init  
+   rosdep update  
+    ```
+For more information, refer [ros wiki](http://wiki.ros.org/)
```
Comment (nkhedekar) [inline docs/_pages/installation.md]:
```suggestion
For more information, refer [ros wiki](http://wiki.ros.org/)

```

File: docs/_pages/installation.md [hunk 3]
```diff
@@ -53,6 +53,20 @@ Supported release is ROS-Melodic. It can be easily installed from official Debia
...
+    
+4. Environment setup
+    ```bash
+    echo "source /opt/ros/melodic/setup.bash" >> ~/.bashrc
+    source ~/.bashrc
+    ```    
+5. Dependencies for building packages
+    ```bash  
+   sudo apt install python-rosdep python-rosinstall python-rosinstall-generator python-wstool build-essential  
+   sudo apt install python-rosdep
```
Comment (nkhedekar) [inline docs/_pages/installation.md]:
```suggestion
```

**Comment by kurshakuz** [comment]
I may also be good to add information about environment setup for both JdeRobot-base and JdeRobot-assets in the same manner

**Comment by nkhedekar** [comment]
Agreed, but I think that should be another issue and pr

**Pull request merged**